GameData/RealismOverhaul/Engine_Configs/U1250_Config.cfg

```diff
@@ -0,0 +1,286 @@
+//	===============================================================
+
+@PART[*]:HAS[#engineType[U1250]]:FOR[RealismOverhaulEngines]
+{
+	%title = U-1250
+	%manufacturer = OKB-2 (Isayev)
+	%description = Small sustainer for MR-1 Meteo sounding rockets, developed in response to the American Aerobee. Pressure-fed. Used after a small solid booster.
+
+	%specLevel = operational
+
+	@MODULE[ModuleEngines*]
+	{
+		%EngineType = LiquidFuel
+	}
+
+	!MODULE[ModuleEngineConfigs],*{}
+	!MODULE[ModuleGimbal] {}
+	!MODULE[ModuleAlternator],*{}
+	!RESOURCE,*{}
+
+	MODULE
+	{
+		name = ModuleEngineConfigs
+		type = ModuleEngines
+		origMass = 0.0147
+		configuration = U-1250
+		modded = false
+		
+		CONFIG
+		{
+			name = U-1250
+			description = Isayev engine that powered the Soviet MR-1 sounding rocket.
+			specLevel = operational
+			maxThrust = 14.417
+			minThrust = 14.417
+			massMult = 1.0
+			heatProduction = 40
+			autoVariationScale = 1.5
+
+			PROPELLANT
+			{
+				name = Kerosene
+				ratio = 0.32714
+				DrawGauge = True
+			}
+
+			PROPELLANT
+			{
+				ratio = 0.67286
+				name = AK20
+			}
+
+			PROPELLANT
+			{
+				name = Nitrogen
+				ratio = 31.5
+				ignoreForIsp = true
+			}
+
+			atmosphereCurve
+			{
+				key = 0 232.1
+				key = 1 204.8
+			}
+
+			IGNITOR_RESOURCE
+			{
+				name = Tonka250
+				amount = 1
+			}
+
+			ullage = True
+			ignitions = 1
+			pressureFed = True
+		}
+		CONFIG
+		{
+			name = U-1700
+			description = Speculative upgrade for the U-1250
+			specLevel = altHist
+			maxThrust = 19.411
+			minThrust = 19.411
+			massMult = 1.0
+			heatProduction = 40
+			autoVariationScale = 1.3
+
+			PROPELLANT
+			{
+				name = Kerosene
+				ratio = 0.32196
+				DrawGauge = True
+			}
+
+			PROPELLANT
+			{
+				ratio = 0.67804
+				name = AK20
+			}
+
+			PROPELLANT
+			{
+				name = Nitrogen
+				ratio = 33.0
+				ignoreForIsp = true
+			}
+
+			atmosphereCurve
+			{
+				key = 0 236.4
+				key = 1 206.5
+			}
+
+			IGNITOR_RESOURCE
+			{
+				name = Tonka250
+				amount = 1
+			}
+
+			ullage = True
+			ignitions = 1
+			pressureFed = True
+		}
+		CONFIG
+		{
+			name = U-2000
+			description = Sustainer engine for the R-112 SAM. Tested, but never entered service.
+			specLevel = operational
+			maxThrust = 23.00
+			minThrust = 23.00
+			massMult = 0.88
+			heatProduction = 40
+			autoVariationScale = 1.2
+
+			PROPELLANT
+			{
+				name = Kerosene
+				ratio = 0.32482
+				DrawGauge = True
+			}
+
+			PROPELLANT
+			{
+				ratio = 0.67518
+				name = AK20
+			}
+
+			PROPELLANT
+			{
+				name = Nitrogen
+				ratio = 34.5
+				ignoreForIsp = true
+			}
+
+			IGNITOR_RESOURCE
+			{
+				name = Tonka250
+				amount = 1
+			}
+
+			atmosphereCurve
+			{
+				key = 0 241.3
+				key = 1 205.6
+			}
+
+			ullage = True
+			ignitions = 1
+			pressureFed = True
+		}
+	}
+
+	RESOURCE
+	{
+		name = Tonka250
+		amount = 1
+		maxAmount = 1
+	}
```

Comment:
Another question out of curiosity, why do we apply an ignition resource *and* set ignitions to 1?  This seems redundant, is there a feature we get from this setup?  (I can see limited ignitor resource and large ignitions count.)

Reply:
The ignition resource is set because the documents said that's what it used to ignite. It doesn't do anything else, since the ignition limit in MEC is set to one.